### PR TITLE
LPS-73633 Implement map modules

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

File: modules/apps/foundation/map/map-common/src/main/resources/META-INF/resources/js/MarkerBase.es.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+ * State definition.
+ * @type {!Object}
+ * @static
+ */
+MarkerBase.STATE = {
+	/**
+	 * Map to be used
+	 * @type {Object}
+	 */
+	map: Config.object().value({}),
```
Comment (carloslancha) [inline modules/apps/foundation/map/map-common/src/main/resources/META-INF/resources/js/MarkerBase.es.js]: Should be alphabetically ordered

File: modules/apps/foundation/map/map-common/src/main/resources/META-INF/resources/js/validators.es.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+/* global HTMLInputElement */
+
+'use strict';
+
+import core from 'metal';
```
Comment (carloslancha) [inline modules/apps/foundation/map/map-common/src/main/resources/META-INF/resources/js/validators.es.js]: Not being used

**Comment by carloslancha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/foundation/map/map-common/src/main/resources/META-INF/resources/js/MarkerBase.es.js (hunk 1 above)
Comment (p2kmgcl) [reply modules/apps/foundation/map/map-common/src/main/resources/META-INF/resources/js/MarkerBase.es.js]: :+1:

File: modules/apps/foundation/map/map-common/src/main/resources/META-INF/resources/js/validators.es.js (hunk 2 above)
Comment (p2kmgcl) [reply modules/apps/foundation/map/map-common/src/main/resources/META-INF/resources/js/validators.es.js]: :+1:

**Comment by carloslancha** [comment]
ci:retest

**Comment by carloslancha** [comment]
Pull request submitted to https://github.com/jbalsas/liferay-portal/pull/806. [See changes here.](https://github.com/carloslancha/liferay-portal/compare/565289c~3...565289c) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carloslancha** [comment]
Pull request submitted to https://github.com/jbalsas/liferay-portal/pull/813. [See changes here.](https://github.com/carloslancha/liferay-portal/compare/630dbc7~7...630dbc7) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**